cli/generators/crud/templates/components/page.js

```diff
@@ -15,7 +15,9 @@ export class <%= ucName %>Page extends Component {
   constructor(props, context) {
     super(props, context);
 
-    autoBind(this);
+    autoBind(this, {
+      bindOnly: ['onClickDetail', 'onClickDelete', 'handleDelete']
+    });
```

Comment:
Just in case it is necessary to be more explicit as possible. However, you can use `autoBind(this)` for this particular case and others only when your are 100% confident that all inner methods will be bound. Following this idea, it is possible in some circumstances when you only want to bind some methods or on the opposite way just won't bind some of them. For instance, there is a component with 5 inner methods (onMethod1, onMethod2, onMethod3, onMethod4, onMethod5)  and you only want to bind three of them so at constructor level you have to declare it on this way: `autoBind(this, {bindOnly: ['onMethod1', 'onMethod2', 'onMethod2']} )` or maybe you don' want to bind one of them so: `autoBind(this, {wontBind: ['onMethod1']} )`